src/System.Diagnostics.Process/tests/ProcessTests.cs

```diff
@@ -359,6 +359,17 @@ public void StartTime_GetNotStarted_ThrowsInvalidOperationException()
             Assert.Throws<InvalidOperationException>(() => process.StartTime);
         }
 
+        [Fact]
+        public void StartTimeIsStable()
+        {
+            DateTime startTime = Process.GetCurrentProcess().StartTime;
+
+            Thread.Sleep(TimeSpan.FromSeconds(5));
```

Comment:
this is a bit long for regular test. 0.5s at most or move it to Outerloop

Reply:
Can do, I'd picked an arbitrary value less than `WaitInMS` used throughout the tests (30 seconds).